.github/config.yml

```diff
@@ -2,13 +2,18 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: >
-  Thanks for opening your first issue here! Be sure to follow the issue template!
+  Thank you for opening your first issue here. :+1:
+  Be sure to follow the issue template.
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: >
-  Thanks for opening this pull request! Please check out our [contributing guidelines](https://endoflife.date/contribute).
-  If you are working on site internals, see https://github.com/endoflife-date/endoflife.date/blob/master/HACKING.md
+  Thank you for opening this pull request. :+1:
+  If you are not familiar with the project, please check out:
+  * [Contributing Guidelines](https://endoflife.date/contribute)
+  * [Guiding Principles](https://github.com/endoflife-date/endoflife.date/wiki/Guiding-Principles)
+  * [Hacking Guide](https://github.com/endoflife-date/endoflife.date/blob/master/HACKING.md) for working on site internals
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Congrats on merging your first pull request!
+  Thank you and congratulations for the first contribution :1st_place_medal: :100: :tada:
+  We are looking for more...
```

Comment:
Unsure about this line, rest all looks great 👍🏽

Reply:
I can remove that. My idea was to engage first time contributors to make other changes in the future.

Reply:
How about this as the second line:

endoflife.date is a community wiki, and we're always looking for more contributions :1st_place_medal: :100: :tada:

Reply:
I find your version good and I changed it, please check...